docs/windows-aliases-and-launchers.md

```
@@ -98,7 +98,6 @@ Like `oc-local`, the environment overrides are scoped to that single invocation.
 ~~~powershell
 oc-provider
 ~~~
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)